### ISSUE TITLE
Community Meeting Documentation

### ISSUE THREAD

**Issue by maxwnewcomer** [openebs/openebs#3816]
This repos `README.md` mentions that 

> OpenEBS holds a monthly community meeting via Zoom on the last Thursday of the month, at 14:00 UTC.
The next meeting is on: Thursday 31 October, at 14:00 UTC
Meeting Link: https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1
Starting in August 2024, the meetings will be recorded and posted on YouTube. [Check here](https://www.youtube.com/@openebscommunity6021)

I just watched the 11/28 community meeting, so this is 1) definitely out of date, 2) doesn't point towards the next community meeting.

Really interested in listening in on the next one and think the docs should be updated to reflect the upcoming schedule.

**Comment by tiagolobocastro** [comment]
Yeah, we need to update it to mention it's on the last thursday and then just provide a calendar.
I think @niladrih was looking into the calendar?

**Comment by maxwnewcomer** [comment]
Following up on this. @tiagolobocastro I believe you mentioned having a meeting the 9th, but I just wanted to follow up so I could make sure to block out time to listen in!

**Comment by tiagolobocastro** [comment]
Hey @maxwnewcomer, we'll get the meeting added tomorrow morning.
@niladrih any progress with the calendar?

**Comment by avishnu** [comment]
@maxwnewcomer, the meeting has been scheduled for Jan 9th Thursday from 14:00 to 14:45 hrs UTC. This is a one-off instance as the scheduled one for Dec couldn't happen due to the year-end vacation.

Details are the same:
Join Zoom Meeting
[https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1](https://www.google.com/url?q=https%3A%2F%2Fus05web.zoom.us%2Fj%2F87535654586%3Fpwd%3DCigbXigJPn38USc6Vuzt7qSVFoO79X.1&sa=D&source=calendar&usd=2&usg=AOvVaw3hgirrMNSl7RMlc3cLi0qV)

Meeting ID: 875 3565 4586
Passcode: f2pN61

**Comment by niladrih** [comment]
I propose that we host an iCalendar file with the following content...
```txt
BEGIN:VCALENDAR
PRODID:-//OpenEBS Maintainers//Online Zoom Meeting//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:PUBLISH
BEGIN:VEVENT
UID:community-meeting-openebs.team@gmail.com
DTSTAMP:20250101T120000Z
ORGANIZER;CN=OpenEBS Maintainers:mailto:openebs.team@gmail.com
DTSTART:20250109T140000Z
DTEND:20250109T150000Z
RRULE:FREQ=MONTHLY;BYDAY=-1TH
RDATE:20250109T140000Z
SUMMARY:Monthly OpenEBS Community Meeting
DESCRIPTION:Join Zoom Meeting: https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1 (meetings are recorded and published on to the OpenEBS Community YouTube channel: https://www.youtube.com/@openebscommunity6021)
LOCATION:Online (Zoom)
END:VEVENT
END:VCALENDAR
```
... on GitHub Pages on the openebs/community repo, and we share the link for it to users.

Try it from this link on my fork of openebs/community: https://niladrih.github.io/openebs-community/community_meeting.ics

**Comment by niladrih** [comment]
The OpenEBS Community meeting is now included in the [CNCF calendar](https://tockify.com/cncf.public.events/detail/837/1738245600000).

**Comment by avishnu** [comment]
@niladrih can you update the latest details and close the issue?
